src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/NewAuthModal.java

```diff
@@ -0,0 +1,54 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+/**
+ * @ownership Social
+ */
+public class NewAuthModal {
+
+  @FindBy(css = ".auth.desktop.signin-page")
+  private WebElement newAuthModal;
+  @FindBy(css = "#loginUsername")
+  private WebElement usernameField;
+  @FindBy(css = "#loginPassword")
+  private WebElement passwordField;
+  @FindBy(css = "#loginSubmit")
+  private WebElement signInButton;
+  @FindBy(css = ".auth-modal iframe")
+  private WebElement iFrame;
+
+  private WebDriver webDriver;
+
+  public NewAuthModal(WebDriver webDriver){
+    this.webDriver = webDriver;
+
+    PageFactory.initElements(webDriver, this);
+  }
+
+  private void switchToFrame(){
```

Comment:
there is no need to create this method
It is one line long and should be used without wrapping into method